packages/plugin-dom-helpers/src/DomHelpers.ts

```diff
@@ -175,6 +175,32 @@ export class DomHelpers<T extends JWPluginConfig = JWPluginConfig> extends JWPlu
         }
         await this.editor.dispatcher.dispatchHooks('@redraw');
     }
+    /**
+     * Prepend a DOM Node to another DOM Node's children.
+     *
+     * @param params
+     */
+    async prepend(container: Node, node: Node): Promise<void> {
```

Comment:
We discussed with @Zinston and the only proper way in english is to say "prepend X to Y" so we would switch the order of the parameters. Same for `append`, obviously.

Reply:
I disagree with the english argument. As we saw with sge and the command insertHTML, it is confusing to have to think about what english rule is for any command. It is far simpler to always set the node that will be modified as the first argument as 99% of our helpers.

Reply:
Well actually either way `node` has to be first... `insertHTML` has the html to insert as first argument so `prepend` should have the node to insert as first argument as well.

Reply:
No, all other command except `insertHTML` has the "target" as the first argument. It's `insertHTML` that should have it's current argument changed.